integrations/gmail.py

```
@@ -0,0 +1,239 @@
+            msg = email.message_from_bytes(email_data)
+
+            # extract email metadata
+            subject = self._decode_header(msg['Subject'])
+            from_addr = self._decode_header(msg['From'])
+            to_addr = self._decode_header(msg['To'])
+            date_str = self._decode_header(msg['Date'])
+
+            # parse the date
+            try:
+                date = email.utils.parsedate_to_datetime(date_str)
+            except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)